src/engine/Operation.cpp

```diff
@@ -131,13 +133,7 @@ shared_ptr<const ResultTable> Operation::getResult(bool isRoot,
             ". This timeout was not caught inside the actual computation, "
             "which indicates insufficient timeout functionality.");
       }
-      // Make sure that the results that are written to the cache have the
-      // correct runtimeInfo. The children of the runtime info are already set
-      // correctly because the result was computed, so we can pass `nullopt` as
-      // the last argument.
-      updateRuntimeInformationOnSuccess(result,
-                                        ad_utility::CacheStatus::computed,
-                                        timer.msecs(), std::nullopt);
+      _runtimeInfo->originalTotalTime_ = timer.msecs();
```

Comment:
I'm not sure if this is equivalent to the code we had before this. This needs further investigation